scripts/generator/__init__.py

```diff
@@ -78,25 +80,33 @@ def sync_from_data_dir(self, data_dir, file_suffix="gpx"):
         if not tracks:
             print("No tracks found.")
             return
+
+        synced_files = []
+
         for t in tracks:
             created = update_or_create_activity(self.session, t.to_namedtuple())
             if created:
                 sys.stdout.write("+")
+                synced_files.extend(t.file_names)
```

Comment:
What if the record already synced before this patch, would the `file_name` be saved in `SYNCED_FILE`?